osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/Components/PathControlPointVisualiser.cs

```diff
@@ -91,6 +93,8 @@ private void onControlPointsChanged(object sender, NotifyCollectionChangedEventA
                         }));
 
                         Connections.Add(new PathControlPointConnectionPiece(slider, e.NewStartingIndex + i));
+
+                        point.Changed += updatePathTypes;
```

Comment:
This subscription seems leaky (likely holds a reference to visualiser instances after they've been disposed). Probably need to double-check if this is safe and unsubscribe from this event for all points in `Dispose()` if not.

Reply:
Binding to `point.Position` breaks the (probably desired) behaviour seen in the bottom gif of [this post](https://github.com/ppy/osu/pull/12122#issuecomment-804190126).

Ended up creating a local bound copy of the `Path.Version` bindable inside `PathControlPointPiece`. This way I can also remove the loop, as all control points will now call this when the path updates.